libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -598,8 +598,8 @@ constructor(
                 .distanceFormatter(distanceFormatter)
 
             // TODO provide a production routing tiles endpoint
-            val tilesUri = URI("")
-            val tilesVersion = ""
+            val tilesUri = URI("https://api-routing-tiles-staging.tilestream.net")
+            val tilesVersion = "2020_02_02-03_00_00"
```

Comment:
This shouldn't be added to the SDK but set up in the desired example. We don't want to ship the staging endpoint directly in the SDK.